addons/portal/views/portal_templates.xml

```diff
@@ -590,12 +593,68 @@
         </t>
     </template>
 
+    <template id="portal_my_users_accesses">
+        <t t-call="portal.portal_layout"><div>
+            <t t-set="page_title" t-value="'Users Accesses'"/>
+            <t t-set="additional_title" t-value="page_title"/>
```

Comment:
Same remark as before for page title / additional title, ... should be defined by controllers + translations + easy to break via editor.